vars/debugReportArchive.groovy

```diff
@@ -0,0 +1,65 @@
+import com.sap.piper.ConfigurationHelper
+import com.sap.piper.DebugReport
+import com.sap.piper.GenerateDocumentation
+import com.sap.piper.Utils
+import groovy.transform.Field
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field def STEP_NAME = getClass().getName()
+@Field Set GENERAL_CONFIG_KEYS = []
+@Field Set STEP_CONFIG_KEYS = [
+    /**
+     * Flag to control whether confidential information (like GitGub credentials) are to be included in the
+     * debug_report.txt. Default value is `false`.
+     * @possibleValues `true`, `false`
+     */
+    'shareConfidentialInformation']
+@Field Set PARAMETER_KEYS = STEP_CONFIG_KEYS + [
+    /**
+     * Flag to enable printing the generated debug_report.txt also to the console.
+     */
+    'printToConsole']
+/**
+ * Archives the debug_report.txt artifact. There is a single config option to enable including confidential
```

Comment:
I would also document 1-3 sentences for what the debug log is for and som examples what will be in there

Reply:
Done.